TRD.md

```
@@ -0,0 +1,47 @@
+# Technical Requirements Document (TRD)
+
+## Scope
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)